src/test/java/test/api/ApiJsonFileTest.java

```diff
@@ -0,0 +1,32 @@
+package test.api;
+
+import baseEntities.BaseApiTest;
+import io.qameta.allure.Description;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+import utils.endpoints.Endpoints;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+public class ApiJsonFileTest extends BaseApiTest {
+
+    @Description("API Post Тест на проверку создания проекта c использованием данных из JSON")
+    @Severity(SeverityLevel.CRITICAL)
+    @Test(testName = "API Post создание проекта c использованием данных из JSON", description = "API Post создание проекта c использованием данных из JSON")
+    public void addProjectUsingJsonFileTest() {
```

Comment:
Данный тест у меня падает с ошибкой при выполнения Parallel.xml
java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)" because "driver" is null

проявлятся ли она у вас?

Если проявляется - как можно это исправить?

Reply:
Да, ошибка появляется случае, есть проект с этим именем передаваемым в json файле уже создан.
Исправляется: либо указать в Jsone иное имя проекта при создании,
                         либо удаление проекта с таким же именем на сайте.